src/drivers/ns16550.rs

```diff
@@ -3,7 +3,6 @@
 
 use crate::kernel_console::Console;
 
-pub const QEMU_VIRT_BASE_ADDRESS: usize = 0x10000000;
 pub const QEMU_VIRT_NS16550_INTERRUPT_NUMBER: u16 = 10;
```

Comment:
As we removed the `QEMU_VIRT_BASE_ADDRESS`, we should also remove `QEMU_VIRT_NS16550_INTERRUPT_NUMBER`

Reply:
Fixed